src/modules/createDao/dialogs/prepareProcessDialog/prepareProcessDialogUtils.api.ts

```diff
@@ -70,6 +70,10 @@ export interface IBuildProcessProposalActionsParams {
      * Address of the execute condition contract if specific permissions are set.
      */
     executeConditionAddress?: Hex;
+    /**
+     * Addresses of the Safe conditions for bodies enabled to create proposals.
+     */
+    safeConditionAddresses?: Hex[];
```

Comment:
```suggestion
     * Addresses of the Safe condition contracts for bodies enabled to create proposals.
     */
    safeConditionAddresses?: Hex[];
```

Maybe this is incorrect, but to match the execute docs above?